style2.css

```diff
@@ -400,4 +452,459 @@
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 5px;
+  margin-right: 8px;
+}
+
+.codekick {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-left: 8px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.github {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.codepen {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.gitlab {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+}
+
+.term {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 120px;
+}
+
+.projects {
+  width: 327px;
+  height: 363px;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 25px;
+  white-space: initial;
+  margin-left: 40px;
+  margin-top: 20px;
+}
+
+.name {
+  height: 48px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5130px;
+}
+
+.name-last {
+  height: 48px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5200px;
+}
+
+.enter {
+  height: 114px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5270px;
+}
+
+.name:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.name-last:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.enter:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.touch-btn {
+  background-color: #36b37c;
+  color: #ebf0ee;
+  font-size: 17px;
+  border-radius: 5px;
+  height: 48px;
+  width: 129px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  position: absolute;
+  top: 5480px;
+  left: 80px;
+}
+
+.touch-btn:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+  cursor: pointer;
+}
```

Comment:
- [ ] [REQUIRED] Please remove `text-decoration: underline rgb(6, 14, 243);` because it causes errors in the design. And please add correct hover color from [FIGMA](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)
- [HINT] Correct colors!

![image](https://user-images.githubusercontent.com/63915024/186478046-a6648c53-9682-4245-9780-7e1a8f01e211.png)

- ### Your Design ❌ 

![image](https://user-images.githubusercontent.com/63915024/186478164-c59af655-7d1a-4859-87bc-b801fb604f3e.png)

Reply:
ok I will remove the underline